### ISSUE TITLE
Sweden: Add production year to SVTPlay when applicable

### ISSUE THREAD

**Issue by Snaah** [retrospect-addon/plugin.video.retrospect#1536]
### Is your feature request related to a problem? Please describe

On Öppet arkiv, the airing date (shown in Retrospect) was when the program/episode was originally broadcast. Now when Öppet arkiv is part of SVT Play, the airing date is when it was added to SVT Play/Öppet arkiv. On svtplay.se they also show the production year. 

### Describe the solution you'd like

Would be great if the airing date shown in Retrospect listings for Öppet arkiv programs/episodes could be changed to the production year. The airing date could moved to the info text instead with "Lades upp:" / "Added:" in front.

One example: [Bamse](https://www.svtplay.se/video/24000071/bamse/bamse-varldens-starkaste-bjorn-sasong-1-skattkartan-1?info=visa) - added to SVT Play/Öppet arkiv 2016-10-28, production year 1966.

In present Retrospect listings, 2016-10-28 is shown. Would be better with 1966.

**Comment by basrieter** [comment]
I understand your request, but that information just is not available from SVT in their listings. It would require fetching each and every video details page for every video in a list. That would take far to long to wait for.

If you have better ways of finding it, I am open for suggestions.

**Comment by Snaah** [comment]
I assumed that information was in SVT's data. Stupid.

Would it be possible (enough speed improvement) if only one episode of a series were scraped? Like for the example Bamse above, the production year is 1966 for all episodes. I figure not as there are hundreds of programs that only have one episode. 

I don't know other ways to get the production year. Anyone else? @Sopor @matsarv

**Comment by basrieter** [comment]
> Would it be possible (enough speed improvement) if only one episode of a series were scraped? Like for the example Bamse above, the production year is 1966 for all episodes. I figure not as there are hundreds of programs that only have one episode.

That could be an option. But I doubt whether this is always the case? There definitely are also shows that span multiple years. And that will thus give incorrect information.

**Comment by Snaah** [comment]
Yes, that is the case for series running over several years, ie [Rederiet, s01e01 1992](https://www.svtplay.se/video/15047372/rederiet/rederiet-sasong-1-farlig-kryssning?info=visa) and [Rederiet, s20e12 (last episode) 2002 ](https://www.svtplay.se/video/15072940/rederiet/rederiet-sasong-20-med-flaggan-i-topp?info=visa). I think having the first production year for all episodes still is better than only when it was added to SVT Play/Öppet arkiv. Or would it be fast enough to scrape one episode for each season?

**Comment by Snaah** [comment]
@basrieter I wrote SVT a while ago. They are going to fix it "in the future", whenever that is. 🤔

 The main page for a series now show production year**s**, like [Rederiet 1992-2002](https://www.svtplay.se/rederiet). If that's in SVT's data, it would be great. If not, scraping production year for first episode of each season? Or just having the first production year for all seasons/episodes while we wait?

**Comment by basrieter** [comment]
But its still not in the list main api list.

**Comment by Snaah** [comment]
Which one(s) are not in the API and which are?

**Comment by basrieter** [comment]
None of them are. The API of the show listing does not contain any of the release years.

**Comment by Snaah** [comment]
Ah, of course. Sorry, my bad.

You said earlier scraping could be an option, but didn't reply to https://github.com/retrospect-addon/plugin.video.retrospect/issues/1536#issuecomment-887827409.
